core/runtime/v2/manager.go

```diff
@@ -481,6 +512,54 @@ func (m *TaskManager) Create(ctx context.Context, taskID string, opts runtime.Cr
 	return t, nil
 }
 
+// tryStreamEvents attempts to stream shim events using bidirectional TTRPC/GRPC channel.
+func (m *TaskManager) tryStreamEvents(ctx context.Context, shim *shimTask) error {
+	// TODO: Use slices.Contains once on Go 1.21
+	supportsStreaming := false
+	for _, f := range shim.Features() {
+		if f == shimbinary.EventStreaming {
+			supportsStreaming = true
+			break
+		}
+	}
+
+	// Not supported, shim will use legacy events reporting.
+	if !supportsStreaming {
+		return nil
+	}
+
+	log.G(ctx).Info("using shim events streaming")
```

Comment:
Does the `ctx` contian info about container ID? This msg log should reflect which container supports streaming.
And one suggestion, since this is for each containers, how about using debug level (info is also ok)?

Reply:
Fixed